### PR TITLE
Anchor: Remove image next to podcast player block

### PR DESCRIPTION
**Pull request by allilevine**
#### Changes proposed in this Pull Request:

* Removes the image next to the podcast player block in the Anchor block, and centers the podcast player

#### Jetpack product discussion

p1612562996197700-slack-C01FD4FGD7X

#### Does this pull request change what data or activity we track or use?

N/A

#### Testing instructions:

* Go to 'wp-admin/post-new.php?anchor_podcast=22b6608&anchor_episode=547d02ae-abde-462d-92af-3de55ed5bcd2&spotify_url=https://open.spotify.com/show/6HTZdaDHjqXKDE4acYffoD?si=EVfDYETjQCu7pasVG5D73Q'
* Check that the player is centered and there is no cover image next to it.
Screen Shot 2021-02-05 at 6 46 09 PM

#### Proposed changelog entry for your changes:

N/A

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js [hunk 1]
```diff
@@ -27,49 +27,24 @@ function spotifyTemplate( { spotifyShowUrl, spotifyImageUrl } ) {
...
-	const { image, guid } = episodeTrack;
+function podcastSection( { episodeTrack, feedUrl } ) {
+	const { guid } = episodeTrack;
 
 	return [
-		'core/columns',
-		{
-			align: 'wide',
-		},
+		'core/group',
```
Comment (pablinos) [inline projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js:34]: Is there a need for the group block? Are we planning on grouping the Spotify button with the player?

**Review by pablinos — COMMENTED** [review]
This is looking good. I wonder if we should make the cover image available in the site's media library, so it can be added if required. That's not something for this PR though!

File: projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js (hunk 1 above)
Comment (allilevine) [reply projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js]: Nope, not that I know of. I'm just new to templates. 🙂 I removed the group block here: https://github.com/Automattic/jetpack/pull/18734/commits/341ce6f28008d9d0920cf017708cb910983143a4

**Comment by allilevine** [comment]
> This is looking good. I wonder if we should make the cover image available in the site's media library, so it can be added if required. That's not something for this PR though!

Is there anything we should leave in place to make adding that easier?

Should this line stay, to pass the image URL to the client side?

https://github.com/Automattic/jetpack/blob/9c794ae32b05475c922ad530b3ef44185dd09acf/projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php#L137

**Comment by pablinos** [comment]
> > This is looking good. I wonder if we should make the cover image available in the site's media library, so it can be added if required. That's not something for this PR though!
> 
> Is there anything we should leave in place to make adding that easier?
> 
> Should this line stay, to pass the image URL to the client side?
> 
> https://github.com/Automattic/jetpack/blob/9c794ae32b05475c922ad530b3ef44185dd09acf/projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php#L137

That's maybe a useful thing to keep, but we'd probably want to trigger something to import the image around that point, and if possible pass the local site version of the URL to the client (in case it gets used in future). I'm not sure of the best mechanism to do that import and we're not using the image at the moment, so I don't think it should hold this up.

**Review by retrofox — APPROVED** [review]
LGTM. 👍 It works as expected.

**Review by jeherve — APPROVED** [review]
Tests well for me. 🚢

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.4` in fae06f6af84e2cba3ee0408120736753988fafc0. I'l let you handle the matching wpcom diff.

**Comment by allilevine** [comment]
r220692-wpcom

**Pull request merged**